src/components/Article.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+export const Article = ({ date, text, title }) => (
+  <React.Fragment>
```

Comment:
Better to use semantic `article` tag here
```suggestion
  <article>
```